store/index.js

```diff
@@ -195,21 +194,43 @@ export default function configureStore(initialState) {
             },
             _stateGetter: (state, key) => {
                 if (key.indexOf('storage:') === 0) {
-                    state.storage = state.storage || {storage: {}};
-                    return state.storage.storage[key.substr(8)];
+                    return state.storage?.storage?.[key.substr(8)];
```

Comment:
Avoid mutating the store at all here.

Reply:
I'm surprised that didn't cause us any problems before since that should've caused it to error out